### DISCUSSION TITLE
links saved in a wierd way normal?

### DISCUSSION THREAD

**Discussion by InfGame** [voidlabs/mosaico#656] [General]
hey! i have a question. 

I have noticed that when you give a word in a text a hyperlink. it is saved in a wierd way. breaking it the 2nd time you edit the template and save it.

for example:

hyperlink example: www.exapledomain.example/examplepage/this-is-an-example/index.html
saved link: ../../this-is-an-example/index.html

my question is if this is normal and if there is any way to turn it off.

**Comment by bago** [comment]
I guess this is not ok.

Which version of mosaico are you using? 
Can you reproduce it with the last unstable release (0.18.4)?
Can you reproduce it on mosaico.io?
Can you reproduce it on 0.17.5 (last stable release)?

In 0.18.3 we did change few things about tinymce integration and also updated the tinymce version against which we build: so, maybe the bad behaviour have been introduced there.

**Reply by InfGame** [reply]
i am currently using 0.17.4

I have not tried it out on newer versions. since this is the system of my company where i cant really change the version easily

**Reply by InfGame** [reply]
tho we are currently in the process of updating to the newest stable version (0.17.5)

**Reply by bago** [reply]
You didn't tell me if this happens on default mosaico release or just in your deployment.

**Reply by InfGame** [reply]
according to our office it just happens on the productive servers

also when the protocoll is changed from https to http or the domain has a port it seems to work just fine.

**Reply by bago** [reply]
I can only investigate issues that can be reproduced with a clean release of mosaico. If this happens only in your deployment then I can't do anything. Maybe you simply changed it/configured it in a way that is wrong.

So, first thing: provides steps to reproduce the issue with a clean mosaico release.

**Reply by InfGame** [reply]
i will try to reproduce it. and will send you the steps of how i did it if i can

**Reply by InfGame** [reply]
steps to recreate the issue:

- make new template
- edit text (with a hyperlinked word)
- save the template
- reopen it
- save it again
- use the template in an email and send it
- try to click the hyperlinked word in the text

Happened on the newest stable release 0.17.5 too.

**Reply by bago** [reply]
I'm sorry I can't see the issue.

I just downloaded mosaico 0.17.5 from github
- npm install
- grunt
- start a new versafix-1 template
- add a sitearticle block
- link a word with the link "http://www.google.com/something"
- link another word with the link "https://www.google.com/somethingelse"
- save
- close
- reopen the template
- save
- send a test
- both links works

**Reply by InfGame** [reply]
ok that was expected. tho can i ask what the links looked like when you reopened the template and looked at it?

**Reply by bago** [reply]
they are the same as the ones I wrote:
http://www.google.com/something and https://www.google.com/somethingelse

Are you sure you are checking a clean mosaico 0.17.5 install? If so, please provide more delailed reproducing steps: maybe your links have something special?

**Reply by InfGame** [reply]
our links are fairly long. so that might be a difference. bc we noticed that shorter links dont get compressed int these ../ sections that often

**Reply by bago** [reply]
I tried with 
http://www.google.com/somethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomethingsomething and https://www.google.com/somethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelsesomethingelse
and they still work.

**Reply by InfGame** [reply]
kinda ment ore devided like http://www.google.com/something/something/something/something/something/something/etc

**Comment by InfGame** [comment]
This issue is happening again on the latest stable mosaico version 0.18.10

**Reply by bago** [reply]
Well, I don't know what to say.

The last time I've never reproduced your issue, so I'm surprised it changed in any way between versions.

Can you reproduce with unchanged mosaico? Please provide me step by step from the checkout from github what I have to do to see the issue.
